charts/testkube-logs-service/values.yaml

```diff
@@ -82,7 +82,7 @@ api:
     # -- Agent gRPCS port
     agentPort: 8443
     # -- TLS secret name which contains the certificate files TODO(emil): conflicts with agent ingress
```

Comment:
Please remove this TODO since the issue will be fixed now.

Reply:
done